cmd/edit.go

```diff
@@ -35,30 +32,32 @@ func newEditProjectCmd() *cobra.Command {
 
 	editProjectCmd := &cobra.Command{
 		Use:   "edit",
-		Short: "This command will edit the configuration of the PROJECT file",
-		Long:  `This command will edit the configuration of the PROJECT file`,
+		Short: "This command will edit the configuration of the configuration file",
+		Long:  `This command will edit the configuration of the configuration file`,
 		Example: `
 		# To enable the multigroup layout/support
 		kubebuilder edit --multigroup
 		
 		# To disable the multigroup layout/support
 		kubebuilder edit --multigroup=false`,
 		Run: func(cmd *cobra.Command, args []string) {
-			dieIfNoProject()
+			util.DieIfNotConfigured()
 
-			projectInfo, err := scaffold.LoadProjectFile("PROJECT")
+			projectConfig, err := config.LoadFrom(config.DefaultPath)
 			if err != nil {
-				log.Fatalf("failed to read the PROJECT file: %v", err)
+				log.Fatalf("failed to read the configuration file: %v", err)
 			}
 
-			if projectInfo.Version != project.Version2 {
-				log.Fatalf("kubebuilder multigroup is for project version: 2, the version of this project is: %s \n", projectInfo.Version)
-			}
+			if opts.multigroup {
+				if projectConfig.Version != project.Version2 {
+					log.Fatalf("kubebuilder multigroup is for project version: 2, the version of this project is: %s \n", projectConfig.Version)
+				}
 
-			// Set MultiGroup Option
-			projectInfo.MultiGroup = opts.multigroup
+				// Set MultiGroup Option
+				projectConfig.MultiGroup = true
+			}
 
-			err = saveProjectFile("PROJECT", &projectInfo)
+			err = projectConfig.SaveTo(config.DefaultPath)
```

Comment:
```suggestion
			err = projectConfig.SaveProjectFile()
```
The `saveProjectFile("PROJECT", &projectInfo)` make clear that it will persist the file. 
However, `projectConfig.SaveTo(config.DefaultPath)` makes not. Also, IHMO it should be something as:

```
err : = projectConfig.saveProjectFile()
```

PS.: The config.DefaultPath does not need to be passed arg. Note that the file is always saved and load from/in the default path.

Reply:
Why does `projectConfig.SaveTo(PATH)` doesn't make it clear?

PS: I will create a default function shortcut the same way I will create a load default, but the other method will stay for things like tests and so on.

Reply:
The project file is saved always in the same place with the same name. So, we should NOT allow it to be done in another way in order to avoid mistakes and issues. E.g someone by mistake to change it. In this way, I  do not think that it should receive an arg at all. 

Then, we would have:
`projectConfig.Save()`

IHMO: the above is acceptable.

Reply:
Right, storing where i was loaded from and saving to the same places sounds like a good idea.